src/Resources/config/services.xml

```diff
@@ -5,6 +5,11 @@
         <import resource="services/*.xml" />
     </imports>
 
+    <parameters>
+        <parameter key="default_logo_file">@SyliusInvoicingPlugin/Resources/assets/sylius-logo.png</parameter>
+        <parameter key="sylius.invoicing.logo_file">%env(default:default_logo_file:SYLIUS_INVOICING_LOGO_FILE)%</parameter>
```

Comment:
Can we add `template` to the parameter name?
```suggestion
        <parameter key="sylius.invoicing.template.logo_file">%env(default:default_logo_file:SYLIUS_INVOICING_LOGO_FILE)%</parameter>
```

Also, for now I'm not convinced with `default_logo_file` parameter as a fallback. IMHO, we should go with only one argument.

Reply:
done.
The thing is, the env-processor accepts only another parameter name for the default clause.
In order to avoid b/c breaks, (because of missing env variable in end user projects), I introduced the ```default_logo_file``` parameter, so that this current behavior is not changed, but can be altered to users needs.

The ```default``` processor was only introduced in Symfony 4.3, so there may be unwanted b/c breaks already - but since Sylius 1.6 is not locked to a specific Symfony 4 version, I felt this was acceptable.